src/GeositeFramework/js/Plugin.js

```diff
@@ -401,6 +405,10 @@ require(['use!Geosite',
             view.$uiContainer = $uiContainer;
 
             $uiContainer
+                // Call `hibernate` method on plugin when the eye icon is clicked
+                .find('.plugin-eye').on('click', function () {
+                    model.hibernate();
+                }).end()
```

Comment:
This change looks good, but considering that the existing framework (on `develop`) already has a "minimize" button which causes the plugin to hibernate, I'm wondering if this is necessary. It's possible that some code became orphaned while implementing the prototype. Or maybe the "minimize" code was deleted?